code/game/dna/mutations/mutation_powers.dm

```diff
@@ -800,7 +800,7 @@
 			M.change_gender(FEMALE)
 
 	if(eyes_organ)
-		var/new_eyes = input("Please select eye color.", "Character Generation", eyes_organ.eye_color) as null|color
+		var/new_eyes = tgui_input_color(user, "Please select eye color.", "Character Generation", eyes_organ.eye_color)
 		if(new_eyes)
 			M.change_eye_color(new_eyes)
```

Comment:
```suggestion
		var/new_eyes = tgui_input_color(user, "Please select eye color.", "Character Generation", eyes_organ.eye_color)
		if(isnull(new_eyes))
			return
		M.change_eye_color(new_eyes)
```
It is much more reliable to use isnull() check in tgui inputs
I personally have been burned more than once by neglecting isnull in this case, don't repeat my mistakes :D
Please replace everywhere after the check inputs with `if(isnull()) return`